pkg/webhook/webhook.go

```diff
@@ -331,6 +332,31 @@ func getNetworkAttachmentDefinition(namespace, name string) (*cniv1.NetworkAttac
 	return &networkAttachmentDefinition, nil
 }
 
+func parseNetSelectionResourceRequest(net *multus.NetworkSelectionElement, reqs map[string]int64) (map[string]int64, error) {
+	/* for each network in annotation ask API server for network-attachment-definition */
+	networkAttachmentDefinition, err := getNetworkAttachmentDefinition(net.Namespace, net.Name)
+	if err != nil {
+		/* if doesn't exist: deny pod */
+		reason := errors.Wrapf(err, "could not find network attachment definition '%s/%s'", net.Namespace, net.Name)
+		glog.Info(reason)
```

Comment:
good opportunity to up the log level here from Info?

Reply:
Good catch! Changed to glog.Error